Dfe.Data.SearchPrototype/Web/Mappers/FacetsAndSelectedFacetsToFacetsViewModelMapper.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Dfe.Data.SearchPrototype.Common.Mappers;
+using Dfe.Data.SearchPrototype.SearchForEstablishments.Models;
+using Dfe.Data.SearchPrototype.Web.Models;
+
+namespace Dfe.Data.SearchPrototype.Web.Mappers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed class FacetsAndSelectedFacetsToFacetsViewModelMapper : IMapper<FacetsAndSelectedFacets, List<Facet>?>
```

Comment:
It would make more sense to rename models in mid tier to match the view model. Example: EstablishmentFacet to Facet and FacetResult to FacetValue

Reply:
Thanks for offering to do that Asia :)